### ISSUE TITLE
manual setlist entry

### ISSUE THREAD

**Issue by nkannath** [Discord-Phingo/Phingo#2]
Ability to manually edit the setlists instead of relying on pulling from Twitter, Phish.net, etc.

**Comment by JonathanCooper** [comment]
Would this rely on having an "admin" type role that has elevated permissions?

**Comment by nkannath** [comment]
Yes, I imagine we would need a few admins to maintain the site. Maybe also have the ability to modify scores if some issue were to occur.  For instance, last games we had people write "curtain" instead of "the curtain" or "the curtain with". We decided to award tentative points to them. This exact instance can be mitigated if we were to limit song choices to a dictionary, though.
